### PR TITLE
Aws instance scheduler

### PR DESCRIPTION
**Pull request by maximrozin**
In a comment on the pull request, include the following information and check off the list.

 - [x] Mini bio
  - Added
 - [x] Desired date of publishing
  - 2019-02-25
 - [ ] Social media contact info
 - [x] Redaction: yes
  - Already reviewed and edited by Brendan Keuning and Tracy Hamilton
 - [x] Comments: no

@catlook Please review.

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-4f34b1cd3a/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-4f34b1cd3a/)

File: _posts/2019-02-25-dramatically-cut-costs-with-aws-instance-scheduler.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+ 
+Instance Scheduler supports:
+* Stopping and starting stand-alone EC2 instances
+* Stopping and starting RDS instances
+* Stopping and starting instances in multiple regions
+* Starting or stopping instances manually outside the defined schedule
+* Partial automation (stop-only or start-only schedule)
+
+It does not support:
+* Stopping or starting instances that are part of an auto-scaling group (native ASG scheduled actions can be used in this case)
```
Comment (catlook) [inline _posts/2019-02-25-dramatically-cut-costs-with-aws-instance-scheduler.md]: add blank line above the bullets

File: _posts/2019-02-25-dramatically-cut-costs-with-aws-instance-scheduler.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+### How it works
+
+A DynamoDB table contains definitions of Periods and Schedules, with every Schedule having a unique name. Resource tags are used to associate EC2 and RDS instances with a particular Schedule (e.g. Schedule=uk-office-hours, when “uk-office-hours” is a name of one of the Schedules we defined in DynamoDB). 
+
+So what are Periods? Periods contain conditions that define the hours/days/months an instance will run. Schedules are composed of one or more Periods.
+
+For example:
+
+Assume that instances need to run during office hours (Monday-Friday 9am-5pm ET) and also perform batch processing every night 2am-4am ET. To achieve this, we will define 2 periods:
```
Comment (catlook) [inline _posts/2019-02-25-dramatically-cut-costs-with-aws-instance-scheduler.md]: ... we define ...

File: _posts/2019-02-25-dramatically-cut-costs-with-aws-instance-scheduler.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+![Periods]({% asset_path 2019-02-25-dramatically-cut-costs-with-aws-instance-scheduler/aws-instance-scheduler-schedule.png %})
+The Schedule “office-and-batch” contains the two running Periods we defined (“office-hours” and “batch”) and specifies the time zone. With the Schedule defined, we can associate our EC2 and RDS instances with it by adding a resource tag with a tag key of “Schedule” and a tag value of “office-and-batch.” Note that the tag value is identical to the name of the Schedule we defined.
+
+### How much does Instance Scheduler cost? 
+The solution itself is free, however, since it utilizes a few AWS services under the hood (DynamoDB, Lambda, CloudWatch), and is deployed in your AWS account, you will still pay for these services. But, in our experience, unless you have a significantly large AWS environment, these costs are generally below $10 per month — and we’ve yet to see a customer that doesn’t save at least $10 in usage costs with Instance Scheduler. Essentially, it pays for itself.
```
Comment (catlook) [inline _posts/2019-02-25-dramatically-cut-costs-with-aws-instance-scheduler.md]: ... however, because it ...

**Comment by catlook** [comment]
Pending edits and delayed for publish until Feb 25.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-80537b9940/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-cd28b07999/)

**Pull request merged**